public/assets/inst_src.js

```diff
@@ -131,12 +131,14 @@ function runInstruction(raw) { //, RISCV) {
         // I-TYPE, opcode: 0b0010011
         case 0x13:
             var funct3 = inst.get_funct3();
+            RISCV.instruction_amounts['arithmetic'] = RISCV.instruction_amounts['arithmetic'] + 1;
             switch(funct3) {
                 
                 // ADDI
                 case 0x0:
                     RISCV.gen_reg[inst.get_rd()] = RISCV.gen_reg[inst.get_rs1()].add(signExtLT32_64(inst.get_I_imm()));
                     RISCV.pc += 4;
+                    //RISCV.instruction_amounts.set('arithmetic', RISCV.instruction_amounts.get('arithmetic') + 1);
```

Comment:
why comment out here?

Reply:
so i think this file generates inst.js, or vice versa, but same reason as above